modules/arm_plugin/thirdparty/CMakeLists.txt

```diff
@@ -64,9 +64,12 @@ else()
         examples=0
         internal_only=0
         build_dir=${CMAKE_CURRENT_BINARY_DIR}
-        -j8
     )
 
+    if (DEFINED ARM_COMPUTE_SCONS_JOBS)
```

Comment:
please, rebase and add `ARM_COMPUTE_SCONS_JOBS` option to https://github.com/openvinotoolkit/openvino_contrib/blob/master/modules/arm_plugin/cmake/features.cmake

Reply:
I've added `ARM_COMPUTE_SCONS_JOBS` and `ARM_COMPUTE_WERROR` to `features.cmake`. However I doubt whether it's necessary to add both options to `IE_OPTIONS`. Especially `ARM_COMPUTE_WERROR` one